pom.xml

```diff
@@ -281,8 +281,25 @@
       <version>${jersey.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.kotlintest</groupId>
+      <artifactId>kotlintest-runner-junit5</artifactId>
+      <version>3.4.2</version>
```

Comment:
`Kotlintest` changed names to `kotest` and has a newer version, can we use that here?  I plan on upgrading other places soon.